pkg/network/tracer/tracer_test.go

```diff
@@ -1529,14 +1528,17 @@ func TestHTTPSViaOpenSSLIntegration(t *testing.T) {
 	serverDoneFn := testutil.HTTPServer(t, "127.0.0.1:443", enableTLS)
 	defer serverDoneFn()
 
+	// Run wget once to make sure the OpenSSL is detected and uprobes are attached
+	exec.Command(wget).Run()
```

Comment:
Check error here?

Reply:
I believe this will always return an error:
```
$ wget
wget: missing URL
Usage: wget [OPTION]... [URL]...

Try `wget --help' for more options.
$ echo $?
1
```
And from the documentation of `Run()`:
> The returned error is nil if the command runs, has no problems copying stdin, stdout, and stderr, **and exits with a zero exit status**.